rest/admin_api.go

```diff
@@ -642,6 +645,8 @@ func (h *handler) handlePutDbConfig() (err error) {
 	})
 	if err != nil {
 		base.WarnfCtx(h.ctx(), "Couldn't update config for database - rolling back: %v", err)
+		// set runtime cfgCas to 0 to force a reload of db config from the bucket
+		h.server.forceRuntimeConfigReload(contextNoCancel, dbName)
```

Comment:
I think I'm OK with this fix, but I think there's a race that exists here and I think we want to pass this value to `fetchAndLoadDatabase`.

`forceRuntimeConfigReload` -> acquires `sc.Lock`, changes `sc.dbconfig`, then `fetchAndLoadDatabase` reacquires the lock.

I think it would be _really_ unlikely for someone to come in between these two steps but it could be avoided by adding a parameter to `fetchAndLoadDatabase`. I also think that would be more conceptually clear what is going on, though I do see it would have to be passed down a few levels of functions.